src/Examine.Lucene/Suggest/IndexReaderReference.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using Lucene.Net.Index;
+
+namespace Examine.Lucene.Suggest
+{
+    /// <summary>
+    /// Reference to an instance of an IndexReader on a Lucene Index
+    /// </summary>
+    public class IndexReaderReference : IIndexReaderReference
+    {
+        private readonly ReaderManager _readerManager;
+        private bool _disposedValue;
+        private DirectoryReader _indexReader;
+
+        public IndexReaderReference(ReaderManager readerManager)
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done